tests/unit/recommenders/utils/test_gpu_utils.py

```diff
@@ -46,6 +46,11 @@ def test_get_cudnn_version():
     assert get_cudnn_version() > "7.0.0"
 
 
+@pytest.mark.gpu
+def test_cudnn_enabled():
+    assert torch.backends.cudnn.enabled == True
```

Comment:
`assert torch.backends.cudnn.enabled` would also work here.

Reply:
The problem with just using assert is that it also takes other values. For exampe if you have something like this:
```
def test_simple():
    assert 1
    assert True
    assert "a"
    assert False
```
you'll get 3 oks and one error.

Reply:
That is correct, in this scenario `assert torch.backends.cudnn.enabled` can only be `True` or `False`, so we still get the desired behavior in either cases. But I agree, your way is better for being more explicit and readable. Dropping `==True` is only more concise.